airgun/views/package.py

```diff
@@ -0,0 +1,92 @@
+
+from widgetastic.widget import Checkbox, Select, Text, View
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import (
+    BaseLoggedInView,
+    ReadOnlyEntry,
+    SatTab,
+    SatTable,
+    SearchableViewMixin,
+)
+from airgun.widgets import ItemsList
+
+
+class SearchablePackageViewMixin(SearchableViewMixin):
```

Comment:
I would say that is a little bit redundant class, maybe you can use similar approach like here:
https://github.com/SatelliteQE/airgun/blob/master/airgun/views/errata.py#L33

Reply:
No problem I can do that , but still think that this should be a separate widget.

Reply:
they still are different (installable vs upgradable), otherwise we can think about uniting them into generic widget. But once more, it is valid for some generic situation, not separate View for single situation without any re-use. Just thoughts